src/shell.zig

```diff
@@ -302,9 +356,14 @@ pub fn exec_status_ok(shell: *Shell, comptime cmd: []const u8, cmd_args: anytype
 
     try expand_argv(&argv, cmd, cmd_args);
 
+    // TODO(Zig): use cwd_dir once that is available https://github.com/ziglang/zig/issues/5190
+    var buffer: [std.fs.MAX_PATH_BYTES]u8 = undefined;
+    const cwd_path = try shell.cwd.realpath(".", &buffer);
+
```

Comment:
Not happy about all these copy-paste here, but also not sure if its worth to try to dry it out -- it's hard to move stack-allocated `bufer` to a helper. Hopefully, `cwd_dir` will be available in the next release.